examples/ensemble_example/client.py

```diff
@@ -0,0 +1,58 @@
+import argparse
+from pathlib import Path
+from typing import Dict, Tuple
+
+import flwr as fl
+import torch
+import torch.nn as nn
+from flwr.common.typing import Config
+from torch.nn.modules.loss import _Loss
+from torch.optim import Optimizer
+from torch.utils.data import DataLoader
+
+from examples.models.ensemble_cnn import ConfigurableMnistNet
+from fl4health.clients.ensemble_client import EnsembleClient
+from fl4health.model_bases.ensemble_base import EnsembleModel
+from fl4health.utils.load_data import load_mnist_data
+from fl4health.utils.metrics import Accuracy
+from fl4health.utils.sampler import DirichletLabelBasedSampler
+
+
+class MnistEnsembleClient(EnsembleClient):
+    def get_data_loaders(self, config: Config) -> Tuple[DataLoader, DataLoader]:
+        batch_size = self.narrow_config_type(config, "batch_size", int)
+        sampler = DirichletLabelBasedSampler(list(range(10)), sample_percentage=0.75)
+        train_loader, val_loader, _ = load_mnist_data(self.data_path, batch_size, sampler=sampler)
+        return train_loader, val_loader
+
+    def get_model(self, config: Config) -> nn.Module:
+        ensemble_models: Dict[str, nn.Module] = {
```

Comment:
Super unimportant, but do you need the type annotation for this dictionary?

Reply:
yeah my bad, I meant to mention that its throwing a mypy error if I don't do the type cast with the Dict[str, nn.Module]

Reply:
Interesting. Just wanted to check 🙂